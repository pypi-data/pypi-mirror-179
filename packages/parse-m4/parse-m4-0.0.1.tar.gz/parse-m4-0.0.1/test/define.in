define(foo,bar)
foo foo
foofoo
foo`foo'foo
define(bar,xyz)
foo foo
foofoo
foo`foo'foo
define(abc, 123)
abc abc
define( zzz, xxx )
zzz zzz
