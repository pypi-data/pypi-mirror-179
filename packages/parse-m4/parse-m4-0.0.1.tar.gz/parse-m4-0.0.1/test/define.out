
bar bar
foofoo
barfoobar

xyz xyz
foofoo
xyzfooxyz

123 123

xxx  xxx 
